.github/auto_request_review.yml

```diff
@@ -1,11 +1,11 @@
 # More info at https://github.com/necojackarc/auto-request-review
 reviewers:
-  # The default reviewers
   defaults:
-    # Example of Github Team. Github team must have write access to repo.
-    # NOTE: This assigned the team itself, not members of the team. The Github
-    # team auto PR assignment will then turn this into individuals
-    - team:celestia-core # This is the Github Team
+    - cmwaters
```

Comment:
@cmwaters LMK if you would like to be removed from this list. If that's the case, we can explicitly add you to PRs that need your attention

Reply:
I'm okay with this for now. I'm not too sensitive to github notifications